torch/testing/_internal/common_methods_invocations.py

```diff
@@ -143,6 +143,45 @@ def close_to_int(x, eps=0.1):
     return (y < eps) | (y > (1 - eps))
 
 
+def sample_inputs_slice(op_info, device, dtype, requires_grad, **kwargs):
+    for i in [
```

Comment:
Just yield each SampleInput so this avoids stuffing them into a list and enumerating that list